src/Service.GraphQLBuilder/GraphQLUtils.cs

```diff
@@ -182,10 +182,13 @@ public static bool TryExtractGraphQLFieldModelName(IDirectiveCollection fieldDir
             {
                 if (dir.Name.Value == ModelDirectiveType.DirectiveName)
                 {
-                    dir.ToObject<ModelDirectiveType>();
-                    modelName = dir.GetArgument<string>(ModelDirectiveType.ModelNameArgument).ToString();
+                    ModelDirectiveType modelDirectiveType = dir.ToObject<ModelDirectiveType>();
```

Comment:
nit: from the PR description, please remove your branch name and make it more readable - the same description goes into release notes so it should be understandable to our developer consumers.